### ISSUE TITLE
Tree view column sorting icons are reversed

### ISSUE THREAD

**Issue by spire-mike** [odoo/odoo#68318]
Impacted versions:
13

Steps to reproduce:
Load any tree view with sortable columns. Click one of the columns to sort it. The sorting arrow icon points down, indicating descending sort, but the values are sorted in ascending order. Click the column again, the arrow points up indicating ascending sort, but the values are sorted in descending order.

Current behavior:
Item sort direction and arrow icon direction do not match.

Expected behavior:
Item sort direction and arrow icon direction should match.

Video/Screenshot link (optional):
This is an ascending sort, but arrow points down:
![image](https://user-images.githubusercontent.com/77629408/112373332-3883d600-8cb7-11eb-83d1-2ae3d3001706.png)

This is a descending sort, but arrow points up:
![image](https://user-images.githubusercontent.com/77629408/112373378-46395b80-8cb7-11eb-8140-d64830646135.png)

Support ticket number submitted via odoo.com/help (optional):

**Comment by pedrobaeza** [comment]
I see correct the icons: the down arrow means that dates start from the lowest ones to the highest, and vice versa. @Yenthe666 do you agree with me?

**Comment by Yenthe666** [comment]
Yep, same for me :)

**Comment by pedrobaeza** [comment]
In Odoo it has always been the contrary, so you have to learn new way of interpret this :wink:

**Comment by spire-mike** [comment]
@pedrobaeza that's not true. Check my Odoo 10 screenshots.

**Comment by Yenthe666** [comment]
Let's ask our holy decision maker @mart-e to give the final verdict :)

**Comment by spire-mike** [comment]
@Yenthe666 - ok. Just to add one thing, I think it's pretty universally accepted that a down arrow means "descending" and an up arrow means "ascending". If we agree on this, then the current functionality is backwards, because a down arrow will result in a sort of [0, 1, 2, 3, 4] and an up arrow will result in [4, 3, 2, 1]. This is backwards. Compare it to Odoo 10 and you will see there it is correct. I think somebody reversed something at some point since Odoo 10, and nobody ever noticed.

**Comment by pedrobaeza** [comment]
There's no universal agreement:

https://ux.stackexchange.com/questions/2041/i-want-to-let-the-user-toggle-a-column-heading-to-sort-a-grid-by-a-date-field-h
https://ux.stackexchange.com/questions/37564/use-up-or-down-arrow-to-represent-sort-ascending-at-table-header
https://stackoverflow.com/questions/338166/which-direction-should-the-arrows-point-in-a-sorted-table

For me, it's more UX as is right now.

**Comment by mart-e** [comment]
Hi,

I don't really have an opinion on that. Maybe we should track down where it changes to see if it was intentional ?

**Comment by spire-mike** [comment]
This is the call to `search_read` in v10 when the arrow is pointing down:

![image](https://user-images.githubusercontent.com/77629408/112637396-7e09e580-8e14-11eb-8566-540cb4e11022.png)

This is the same call in v13:

![image](https://user-images.githubusercontent.com/77629408/112637331-6894bb80-8e14-11eb-958b-ce61dff39470.png)

I have never seen the arrow point up for descending anywhere in the real-world besides Odoo 13. I'm not saying that means it's universally excepted, but I've never seen it used in practice this way. In my opinion the links referenced above by @pedrobaeza heavily lean towards a down arrow meaning descending. There is lots of discussion about it, but most comments indicate down means descending. Here are some examples in common libraries. All of which use a down arrow to mean descending:

https://material-ui.com/components/tables/
https://vuetifyjs.com/en/components/data-tables/
https://material.angular.io/components/sort/overview
https://examples.bootstrap-table.com/

This screenshot is from material design website: https://material.io/components/data-tables#anatomy

![image](https://user-images.githubusercontent.com/77629408/112640779-0b026e00-8e18-11eb-9fa7-27d30265db44.png)

**Comment by mart-e** [comment]
With a bit of search, it seems that the 9.0 in community was one behaviour and the enterprise was the othe. In 10.0, the code of the web client was unified and we used the second one. So it looks like @ged-odoo introduced the new behaviour but don't know if we should change it.

**Comment by Yenthe666** [comment]
Let's reopen it until @ged-odoo gives his feedback as to why he did it.

**Comment by ged-odoo** [comment]
ooh, I vaguely remember some debates about that. Not sure about the details, but we definitely checked what many other programs did, and we made a decision.   I personally think that the triangle points to a direction, so it should be increasing (meaning that the current state is correct). Also, I don't think that we have had a lot of feedback on this issue for the last few years, so either most people are happy with it, or don't care that much.

I just forwarded this issue to our product owner, so he will be aware of your wish.
